demo/cache/cache-controller.js

```diff
@@ -28,4 +29,26 @@ app.controller('CacheCtrl', function($scope, $rootScope, FDP, Caching, Log) {
       Log.appendToLog(logMsg);
     });
   };
+  
+  $scope.cacheSearch = function(url) {
+    Log.reset();
+    Log.setLogElementId('log-panel');
+    $scope.isCachingStarted = true;
+    var logMsg = "Caching started";
+    Log.appendToLog(logMsg);
```

Comment:
Any particular reason for this pattern (creating a variable first and then passing it to a function)? I see you use it more often.

Reply:
I use this function to append new lines to this variable.

Reply:
I don't see any newlines being added [here](https://github.com/rajaram5/linked-data-demo/blob/ccd756174cc21df75ec640b2df6dfb03bad4f7f2/demo/components/log/log-service.js#L9-L23). But that was not what I meant in this comment. My question was: why do you use the following pattern:

```
var x = 'hello';
some_function(x);
```

instead of

```
some_function('hello');
```

I don't see any reuse of the value you assign to `logMsg`.

Reply:
👍